pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1033,9 +1055,9 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 		err                    error
 	)
 	if c.clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
-		// Check internal FSS map.
+		// Check hard coded FSS map for vanilla only
 		c.internalFSS.featureStatesLock.RLock()
-		if flag, ok := c.internalFSS.featureStates[featureName]; ok {
+		if flag, ok := c.releasedVanillaFSS[featureName]; ok {
```

Comment:
Can we first check if the `featureName` exists in the `releasedVanillaFSS` map. If it does, pick the value from there. If not, pick value from `internalFSS`.

Reply:
Done.